templates/delaySlider.html

```diff
@@ -0,0 +1,6 @@
+<form action="#">
+    <p class="outline-range-field">
+        <input type="range" id="outline-delay-slider" min="0" max="1000" value="{{exposeDelay}}" />
+    </p>
+    <p id="current-selected-delay">{{exposeDelay}}</p>
```

Comment:
Could you make the current value also an input field for those that prefer to type the value?

Reply:
I'll try :wink: